documentation/source/getting_started/components_architecture.rst

```diff
@@ -0,0 +1,23 @@
+Components and Architecture
+===========================
+This section includes OpenEBS components and architecture.
+
+Components
+----------
+
+OpenEBS platform contains three major components:
+
+  * Storage PODs or VSMs
+  * An orchestration engine or VSM Scheduler called Maya
```

Comment:
We do not want to talk about VSM

Reply:
Deleted VSM in the first bullet point. Assuming other instances of VSM remain?

Changes will reflect when I push the document to the readthedocs website.